### PR TITLE
Set objectprefix for senosors and removed desired state code

### PR DESCRIPTION
**Pull request by w1ll1am23**
Fixed the object prefixes in some of the sensors (everything from #57) and removed all of the desired state code. There was a TODO item to remove it stating it should be handled by the program using python-wink, I agree.

I also removed the domain directory which stated it was for non API related functions which again should probably be handled by programs using this library.

This also exposes wink_api_fetch which needs called every few hours to keep PubNub updates flowing from the subscription. (Not sure why)

### CODE REVIEW

**Comment by w1ll1am23** [comment]
@bradsk88 if you get a chance, can you take a look at this? Need the wink_api_fetch method available to home-assistant in order to fix an issue with pubnub timing out.

**Comment by bradsk88** [comment]
I'll set aside some time tonight to review it.  There's a good amount of changes here so I want to make sure I understand what's happening.

**Comment by w1ll1am23** [comment]
Makes sense, I don't want to break anything lol. I have been running this I pushed it and haven't run into anything, at least noticeable yet.

File: src/pywink/devices/standard/__init__.py [hunk 1]
```diff
@@ -392,6 +294,7 @@ def available(self):
         connection variable isn't stable.
         Porkfolio can be offline, but updates will continue to occur.
         always returning True to avoid this issue.
+        This is the same for the PorkFolio balance sensor.
```
Comment (bradsk88) [inline src/pywink/devices/standard/__init__.py:297]: I'll merge this but, as a general rule, a comment like this is unnecessary and will likely rot.

**Review by bradsk88 — APPROVED** [review]

**Pull request merged**